Makefile

```diff
@@ -1,15 +1,41 @@
+#!/usr/bin/env bash
+
 # Application
-APP_ROOT := $(CURDIR)
+APP_ROOT := $(abspath $(lastword $(MAKEFILE_LIST))/..)
 APP_NAME := twitcher
 
-# Anaconda
-CONDA := $(shell command -v conda 2> /dev/null)
-ANACONDA_HOME := $(shell conda info --base 2> /dev/null)
-CONDA_ENV := $(APP_NAME)
+# Conda
+CHECK      ?= shell command -v
+CONDA_HOME ?= $($(CHECK) conda info --base 2> /dev/null)
```

Comment:
didn't work: use previous cmd `$(shell conda info --base 2> /dev/null)`

Reply:
This doesn't work for me.

```
tmp:
	@echo "THE RESULT: $(shell conda info --base 2> /dev/null)"
```
```
make tmp
>> THE RESULT: 
```
```
which conda                                                                                                                                                                                                               
>> conda () {
<...>
}
```
** this is the miniconda output on ubuntu

With only ``conda info --base 2> /dev/null`` (without shell), I get the actual conda home path.